addons/uis/apps/android.md

```diff
@@ -29,14 +33,17 @@ The openHAB Android application is a native client for openHAB. The app follows
 ## Getting Started
 
 When first installed the app is in "Demo Mode".
-To connect it to your own openHAB server, first navigate to Settings and uncheck the "Demo Mode" option.
-Normally, after unchecking the Demo Mode, the app will be able to use multicast DNS to autodetect to your openHAB server if it is on the same network.
+To connect to your own openHAB server, first navigate into the Settings menu and uncheck the "Demo Mode" option.
+Normally, after unchecking the Demo Mode, the app will be able to use multicast DNS to auto-detect your openHAB server if it is on the same network.
 
 You also have the option to manually set the server URL in the settings.
 
-**Local server settings:** Please enter the base URL to your openHAB server as you would enter it in the browser to reach the openHAB dashboard.
+**Local server settings:**
+Please enter the base URL to your openHAB server as you would enter it in the browser to reach the openHAB dashboard.
 
-**Remote server settings:** If you want to use openHAB cloud, please enter URL, user and password of you cloud instance here. Please make sure to [secure this connection](http://docs.openhab.org/installation/security.html).
+**Remote server settings:**
+If you want to use openHAB cloud, please enter URL, user and password of you cloud instance here.
```

Comment:
Consider, "If you want to use openHAB cloud, please enter the URL, username and..."